class.tx_jmrecaptcha.php

```diff
@@ -79,12 +99,37 @@ protected function initConfiguration() {
 	 * @return string
 	 */
 	protected function renderNoCaptcha() {
-		$content = '<script type="text/javascript" src="' . htmlspecialchars($this->conf['server']) . '.js?hl=' . htmlspecialchars($this->conf['lang']) . '"></script>';
-		$content .= '<div class="g-recaptcha" data-sitekey="' . htmlspecialchars($this->conf['public_key']) . '" data-theme="' . htmlspecialchars($this->conf['theme']) . '"></div>';
+		$language = $this->getLanguageCode(null);
+		$languageParameter = '';
+		if (!empty($language)) {
+			$languageParameter = 'hl=' . $language;
+		}
+
+		$content = '<script type="text/javascript" src="' . htmlspecialchars($this->conf['server']) . '.js?' . $languageParameter . '"></script>';
```

Comment:
why not `htmlspecialchars($this->conf['server'] . '.js?' . $languageParameter)`?